src/main/scala/cs332/distributedsorting/slave/Slave.scala

```diff
@@ -57,7 +58,8 @@ object Slave {
 
 class Slave private(
   private val channel: ManagedChannel,
-  private val blockingStub: SortingBlockingStub
+  private val blockingStub: SortingBlockingStub,
+  val name :String
```

Comment:
for debugging purpose I add a parameter name to Slave. Because basically Master use ip adress to distinguish two differents slaves. But in local, ip address are the same for all slaves. So instead I use a new parameter name